pkg/audit/manager.go

```diff
@@ -274,62 +275,107 @@ func (am *Manager) auditResources(
 	var errs opa.Errors
 	nsCache := newNSCache()
 
-	for gv, gvKinds := range clusterAPIResources {
-		for kind := range gvKinds {
-			objList := &unstructured.UnstructuredList{}
-			opts := &client.ListOptions{
-				Limit: int64(*auditChunkSize),
+	constraintsList := make(map[string]bool)
+	if *auditMatchKindOnly {
+		constraintList := &unstructured.UnstructuredList{}
+		for _, c := range resourceList {
```

Comment:
resourceList should probably be called something that suggests it's a list of constraint kinds, as before searching through the file to find where it was defined, I thought it was a general list of cluster resources.

Reply:
updated to `constraintsGVK`